README.md

```diff
@@ -16,6 +16,79 @@ DocumentService is a library with the following functions
 - Converts an HTML document to PDF.
 - Replace placeholders in the document with actual string data.
 
+# Basic Usage
+
+## PDF Generation
```

Comment:
Check the conventions that the headings are following. For example, the heading in line no. 8 is "Word document generation". Only the first word starts with a capital letter. The same should be followed to maintain consistency.

Reply:
Made the changes directly. Resolving the comment.